### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Dongni31**
Added a detailed guide on how to properly contribute code to the project through GitHub.

[GitHub Contribution Guide]

### Proposed changes
The `CONTRIBUING.md` file is lacking an informative guide on how to add to the GitHub repository. A simple guide was added to the file for future contributing members to follow. More detail can be added if there is a need.

It is suggested that a streamline method for adding changes is made in the future on top of the changes currently made. This will help maintainers review changes a lot quicker in the future.

### Does this make breaking changes?
No code was changed for this Pull Request. Only documentation changes were made.

### Does this version of Project64 compile and run without issue?
There are no compilation issues related to the changes. All change is documentation related.

### CODE REVIEW

**Comment by DerekTurtleRoe** [comment]
I like this! Looks good to me. 👍

**Comment by Squall-Leonhart** [comment]
if i was a reviewer, i'd ok this.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -8,4 +8,21 @@ Following these guidelines will help make sure that users and developers alike c
 
 Please do NOT post any copyrighted materials. This includes (but is not limited to) ROMs, ROM hacks, IPL and firmware files, etc.
 
-Save states and game saves do not fall under this category, so please post them if they are applicable to your issue!
\ No newline at end of file
+Save states and game saves do not fall under this category, so please post them only if they are applicable to your issue!
+
+# Github Contribution Guide
+
+1. Create a frok of the repository on your own account.
```
Comment (TiagoManuelPC) [inline .github/CONTRIBUTING.md:15]: Should it be fork instead of frok?

Comment (DerekTurtleRoe) [reply .github/CONTRIBUTING.md:15]: Good catch! I didn't notice. 😄 But yes it should be.

**Pull request merged**